test/runtests.jl

```diff
@@ -90,10 +90,7 @@ end
     element B 3
     property list int32 int64 a_list
     end_header
-    0 
-    0 
-    0 
-    """
+    0 \n0 \n0 \n"""
```

Comment:
Same content here, I just reformatted since my editor kept trimming the trailing spaces, causing the test to fail.

Reply:
Nevermind, I reverted this change in b33b662d5a257fb5ea7676a42c9df244362d99ee since it causes julia 1.0 tests to fail, apparently multiline string parsing has changed.